Cargo.toml

```diff
@@ -49,6 +49,7 @@ secp256k1 = { opt-level = 2 }
 secp256k1-zkp = { opt-level = 2 }
 secp256k1-sys = { opt-level = 2 }
 secp256k1-zkp-sys = { opt-level = 2 }
+bitcoin_hashes = { opt-level = 2 }
```

Comment:
I was surprised how long it takes to generate 1000 * `num_tiers` pending blind nonces, and I though there's a bug in my async code or something.

Since then I lowered the default to 100 (should be enough given it's per-amount tier), and I've noticed that `bitcoin_hashes` was not being optimized.

Reply:
1000 blind nonces should be around 4s imo. Was it roughly that long or even more?